rsciio/docstrings.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2022 The HyperSpy developers
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with any project and source this library is coupled.
+# If not, see <https://www.gnu.org/licenses/#GPL>.
+
+FILENAME_DOC = """filename : str
+        Filename of the file to read.
+    """
+
+SIGNAL_DOC = """signal : dict
+        The signal to save.
```

Comment:
`Dictionary containing the signal object.`

Better would be to give more details on the format of the dictionaries, or simply link the API page in the user guide that gives the details on these dictionaries.

Reply:
Indeed, I have left it simple for now thinking that we can improve it in another iteration! :)